src/blas/impl/KokkosBlas3_gemm_impl.hpp

```diff
@@ -517,7 +517,17 @@ struct GEMMImpl {
       ViewTypeBScratch::shmem_size() +
       ViewTypeCScratch::shmem_size();
 
+#if defined(KOKKOS_ENABLE_HIP)
+    // Note lbv, 10/29/20: The LaunchBounds<384,2> leads
+    // to an error with HIP as the heuristics on that platform
+    // yield an optimal_num_blocks=0 which means no ressources
+    // are allocated... Switching to LaunchBounds<384,2> fixes
+    // that problem but I'm not sure if that it a good perf
+    // parameter or why it is set to 2 for Cuda?
```

Comment:
File a Kokkos issue ?

Reply:
Yeah, I'll talk about it with Damien and Daniel, some parameters are inherently different with AMD like the wrap size so this could be something that needs to be different depending on the architecture?